apps/dashboard/src/routes/(navbar-pages)/network-staking/(load-validators)/(load-staking-data)/+layout.ts

```diff
@@ -136,10 +165,32 @@ export const load: LayoutLoad = async ({ depends, parent }) => {
     })
   )
 
-  const totalXrdBalance = derived(accounts, async ($accounts) =>
-    (await Promise.all($accounts.map((a) => getXRDBalance(a.address))))
-      .reduce((acc, curr) => acc.plus(curr), new BigNumber(0))
-      .toString()
+  const totalXrdBalance = derived(
+    [xrdAddress, sharedAccountsEntityDetails],
+    async ([$xrdAddress, $details]) => {
+      if (!$xrdAddress) {
+        return '0'
+      }
+
+      const details = await $details
+      if (details.isOk()) {
```

Comment:
what if it errors? if you don't care about handling it, maybe it's better to use `_unsafeUnwrap()` and let it throw

Reply:
What we should display in `XRD available to be staked:` if it errors in your opinion?

Reply:
it would display nothing and the app would crash - I think that's better than displaying information that is wrong and misleading?